code/modules/medals/rewardsAI.dm

```diff
@@ -0,0 +1,86 @@
+/datum/achievementReward/aicase
+	var/aiskin = "default"
+
+	rewardActivate(mob/activator)
+		if (isAI(activator))
+			var/mob/living/silicon/ai/A = activator
+			if (isAIeye(activator))
+				var/mob/living/intangible/aieye/AE = activator
+				A = AE.mainframe
+			A.coreSkin = aiskin
+			A.update_appearance()
+			return 1
+		else
+			boutput(activator, "<span class='alert'>You need to be an AI to use this, you goof!</span>")
+
+/datum/achievementReward/aicase/bee
+	title = "(AI Core Skin) Bee"
+	desc = "Buzz Buzz!"
+	required_medal = "Bombini is Missing!"
+	aiskin = "bee"
+
+/datum/achievementReward/aicase/ai_ailes
+	title = "(AI Core Skin) Bulky"
+	desc = "Replaces your core with a bulky older model."
+	required_medal = "I'm In"
+	aiskin = "ailes"
+
+/*/datum/achievementReward/aicase/ai_cardboard
+	title = "(AI Core Skin) Cardboard"
+	desc = "Replaces your core with a... Cardboard box. Huh."
+	required_medal = "Tax Haven" //tempted to put it on the "Call 1-800-CODER" medal but that seeeems like a bad idea in practice so QM greentext medal it is
+	aiskin = "cardboard"*/
```

Comment:
?

Reply:
oh that's commented because i was going to reserve that skin for the SICC latejoin core on 15417 rather than having it on a medal

Reply:
i think probably just implement it since it's merged

Reply:
so, delete this?